Backend/dev/migrations/rider-app/1208-add-priority-in-emergency-contact-table.sql

```diff
@@ -0,0 +1,19 @@
+ALTER TABLE atlas_app.person_default_emergency_number ADD COLUMN contact_person_id text;
+ALTER TABLE atlas_app.person_default_emergency_number ADD COLUMN is_following boolean NOT NULL DEFAULT FALSE;
```

Comment:
how will we make already registered emergency contacts enable this as they will have to call updateEmegenyContact API for this ? some nudge at frontend ?

Reply:
Every one have to do the safety setup from front end. in that it will be updated.